lab_1/main.py

```diff
@@ -21,7 +27,15 @@ def remove_stop_words(tokens: list, stop_words: list) -> list or None:
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     """
-    pass
+    if not isinstance(tokens, list) or not tokens:
+        return None
+    if not isinstance(stop_words, list) or not stop_words:
```

Comment:
I think the first part should lead to None, do you agree?

Reply:
Sorry, but isn't it aleading to None already?
As for the 30 and 31 lines.
Just as I need to get in my work if I get incorrect tokens I return None and if I get incorrect stop_words I return tokens without any changes

Reply:
I meant that if stop_words is not a list, you need to return None, if it's an empty list – return tokens
Now you return tokens in both cases